docs/guides/feature_stages.md

```diff
@@ -0,0 +1,39 @@
+[**UDMI**](../../) / [**Docs**](../) / [Guides](#)
+
+# Feature Stages
+
+Sequence tests use the @Feature annotation to tag test functions with their stage of release.
+
+* ALPHA
+    * Definition
+        * The feature (definition, code, schema, tests, etc) is still expected to change, maybe even substantially, while in ALPHA stage.
+        * The feature may not work as expected.
+        * Ready for external testing by manufacturers who are trusted testers/we have a close relationship with.
+        * One CI test needs to pass.
+* BETA
+    * Definition
+        * Development team has finished initial development of feature.
+        * The feature is defined in device qualification plans.
```

Comment:
As per my comment in the doc the UDMI specs can't reference qualification -- the dependency goes the other way.